libraries/api-documenter/.gitignore

```diff
@@ -0,0 +1 @@
+files
```

Comment:
I added some documentation

---
In reply to: [134040711](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134040711) [](ancestors = 134040711)